fastlane/Appfile

```diff
@@ -0,0 +1,6 @@
+# app_identifier("[[APP_IDENTIFIER]]") # The bundle identifier of your app
+# apple_id("[[APPLE_ID]]") # Your Apple Developer Portal username
```

Comment:
Do I need to fill in these details? for it to successfully run?

Reply:
If you are not using Fastlane to push a new build to the AppStore, just this template file is ok I think